pythonScript.py

```diff
@@ -12,6 +12,16 @@
 import sys, os, random, time
 
 if __name__ == "__main__":
+    print "Script depreciated, use instead:"
```

Comment:
Scripts that are purely used for testing new implementations should not be kept in the repo.  You should have those in your gitignore.

Deleting.